### PR TITLE
Removed Mock API from SUI codebase

### PR DESCRIPTION
**Pull request by chalettu**
PV https://www.pivotaltracker.com/story/show/140667939

### CODE REVIEW

**Comment by chalettu** [comment]
@miq-bot add_label fine/no
@miq-bot add_label enhancement

File: package.json [hunk 1]
```diff
@@ -101,6 +100,7 @@
     "karma-sinon": "1.0.5",
     "karma-webpack": "2.0.3",
     "lodash": "3.10.1",
+    "manageiq-api-mock": "https://github.com/ManageIQ/manageiq-api-mock.git#master",
```
Comment (AllenBW) [inline package.json]: prefer the follow: `ManageIQ/manageiq-api-mock#master`

**Comment by chalettu** [comment]
@AllenBW made that change.  Let me know if we can approve this one.

File: package.json (hunk 1 above)
Comment (AllenBW) [inline package.json]: again, the whole url isn't necessary, referencing the user repo and branch or version is all ya need here

prefer the follow: `ManageIQ/manageiq-api-mock#master`

**Comment by AllenBW** [comment]
@chalettu 
the whole url isn't necessary, referencing the user repo and branch or version is all ya need here

prefer the follow: `ManageIQ/manageiq-api-mock#master`

**Comment by chalettu** [comment]
@AllenBW can you review again and approve?

**Pull request merged**